geomstats/geometry/matrices.py

```diff
@@ -22,15 +22,24 @@ def __init__(self, m, n):
         self.metric = MatricesMetric(m, n)
 
     def belongs(self, point):
-        """Check if point belongs to the Matrix space."""
+        """Check if point belongs to the Matrix space.
```

Comment:
"Evaluate if a point ..." to be consistent with other docstrings? :/ Or should we put "Check" everywhere? It's also a detail.

Reply:
I think `Test if a point...` would be best, is has a boolean feel to it.